### PR TITLE
Qf 911 Nuget changes for accessibility and backwards compatibility purposes

### PR DESCRIPTION
**Pull request by PeterDavidWilliams**
The changes here are for updating the views nuget packages to fix some accessibility issues in the assessor service. It also makes that nuget package backwards compatible with .net standard 2.0 since assessor isn't on 6.0 yet

### CODE REVIEW

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.QnA.Api.Views/SFA.DAS.QnA.Api.Views.csproj [hunk 1]
```diff
@@ -17,4 +17,8 @@
             <CopyToOutputDirectory>Always</CopyToOutputDirectory>
         </Content>
     </ItemGroup>
+  <ItemGroup>
```
[the remark this answers is not in the document]
Comment (PeterDavidWilliams) [reply src/SFA.DAS.QnA.Api.Views/SFA.DAS.QnA.Api.Views.csproj]: Condition added

**Review by debrr07 — DISMISSED** [review]

File: src/SFA.DAS.QnA.Api.Views/SFA.DAS.QnA.Api.Views.csproj [hunk 2]
```diff
@@ -17,4 +17,8 @@
             <CopyToOutputDirectory>Always</CopyToOutputDirectory>
         </Content>
     </ItemGroup>
+  <ItemGroup Condition=" $(TargetFramework) == 'netstandard2.0'">
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.QnA.Api.Views/SFA.DAS.QnA.Api.Views.csproj:20]: I removed those packages above and googled the error message - adding the package below and restarting VS removed the error message for me which would be much better if it works
```
<ItemGroup Condition=" $(TargetFramework) == 'netstandard2.0'">
      <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
</ItemGroup>
```

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: pipeline-templates/job/code-build.yml [hunk 3]
```diff
@@ -75,3 +75,8 @@ jobs:
     displayName: 'Publish Artifact'
     inputs:
       PathtoPublish: '$(build.artifactstagingdirectory)/'
+
+  - template: azure-pipelines-templates/build/step/nuget-pack.yml@das-platform-building-blocks
+    parameters:
+      DotNetStandardPackagesToPack: |
+        src/${{ parameters.SolutionBaseName }}.Api.Views/${{ parameters.SolutionBaseName }}.Api.Views.csproj;
```
[the remark this answers is not in the document]
Comment (PeterDavidWilliams) [reply pipeline-templates/job/code-build.yml]: Done

File: src/SFA.DAS.QnA.Api.Views/SFA.DAS.QnA.Api.Views.csproj (hunk 2 above)
Comment (PeterDavidWilliams) [reply src/SFA.DAS.QnA.Api.Views/SFA.DAS.QnA.Api.Views.csproj:20]: Change has been made

**Review by cpwevans — APPROVED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**